src/ChipInput.spec.js

```diff
@@ -4,6 +4,8 @@ import { mount } from 'enzyme'
 import Chip from '@material-ui/core/Chip'
 import ChipInput from './ChipInput'
 
+window.__MUI_USE_NEXT_TYPOGRAPHY_VARIANTS__ = true
```

Comment:
We shouldn't force our users to use the next typography variants.

Reply:
That's totally fair. However, this is just in the test, and without this, I get the following error when running tests.

```bash
Error: Warning: Material-UI: you are using the deprecated typography variants that will be removed in the next major release.
Please read the migration guide under https://material-ui.com/style/typography#migration-to-typography-v2
```

I wasn't able to find any explicit references to the new typography the component.. but I'm probably just missing something.

Reply:
Oh, it's late. You are absolutely right, just keep it. This is a breaking change in Material-UI. We'll update to the non-deprecated prop when we drop 3.x.x compatibility.